fredix1/React-practise
# #161686029 Search through available meals

#### What does this PR do?

 - Enables a user search through available meals

#### Description of Task to be completed?

 - Ensure results reflect users search
- Ensure a user can see the searched meals

#### How should this be manually tested?
 checkout into the branch `ft-search-through-available-meals-161686029`
- Run the application `yarn start`
- You click `see all meals` to see a list of all available meals
- Type into the search box fora meal
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161686029](#161686029)
#### Screenshots (if appropriate)
![search-video-to-gif](https://user-images.githubusercontent.com/31001534/47979178-a7563980-e0d2-11e8-9b61-1423fdf9b8a1.gif)

#### Questions:
